example/spectrogram/app.js

```diff
@@ -14,7 +14,8 @@ document.addEventListener('DOMContentLoaded', function() {
         plugins: [
             WaveSurfer.spectrogram.create({
                 container: '#wave-spectrogram',
-                labels: true
+                labels: true,
+                colorMap
```

Comment:
this seems to be a typo, and should be `colorMap: colorMap`

Reply:
Ah. I'm used to typescript where that would be valid. It actually did work in chrome but I see that it is not guaranteed to work for all versions of javascript.  Fixed.